tests/system/action/motion/test_create.py

```diff
@@ -107,9 +111,37 @@ def test_create_simple_fields(self) -> None:
             },
         )
         self.assert_status_code(response, 200)
-        self.assert_model_exists(
+        motion = self.assert_model_exists(
             "motion/2", {**motion, "attachment_meeting_mediafile_ids": [80]}
         )
+        assert motion.get("submitter_ids") is None
+
+    def test_create_additional_submitter_forbidden(self) -> None:
```

Comment:
Add in this test: the motion being created with submitter and additional submitter.

Reply:
Added it as a separate test